packages/server-wallet/src/handlers/updateChannel.ts

```diff
@@ -0,0 +1,83 @@
+import {Either, left, right, chain, map} from 'fp-ts/lib/Either';
```

Comment:
Minor but most of our files names are using kebab-casing instead of camel casing, so we should probably rename this to `update-channel`?

Reply:
Let's make an issue out of this, and make a lint rule to enforce it?